lib/jazzy/config.rb

```diff
@@ -178,12 +178,9 @@ def hide_objc?
         'sourcekitten.',
       default: []
 
-    config_attr :custom_modules,
-      # command_line: ['-cm', '--build-tool-arguments arg1,arg2,…argN', NArray],
-      description: 'Custom navigation categories to replace the standard ' \
-        "'Classes', 'Protocols', etc. Types not explicitly named " \
-        'in a custom category appear in generic groups at the ' \
-        'end.  Example: https://git.io/v4Bcp',
+    config_attr :modules,
+      description: 'Array of modules that are going to be documented ' \
```

Comment:
This description is unclear to me. I assume it's a list of modules, since array is an implementation concept, and they are not "being documented".
The line with the arguments puzzles me too, but maybe I need to dive into the tool more first.